Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -373,15 +373,21 @@ div.bocList div.bocListTableBlock div.bocListNavigator a:not([href]) img
 /*endregion*/
 
 /*region validation marker*/
-div.bocList .bocListCellStructureElement > .validationErrorMarker
-{
-  margin-top: calc(var(--remotion-themed-spacing) / 2);
-}
-
 div.bocList .bocListCellStructureElement > .validationErrorMarker,
 div.bocList .bocListContent > .validationErrorMarker
 {
+  padding: calc(var(--remotion-themed-spacing) / 2);
   border: var(--remotion-themed-border-placeholder);
+  border-radius: var(--remotion-themed-border-radius);
+}
+
+div.bocList .bocListCellStructureElement > .validationErrorMarker:focus,
+div.bocList .bocListContent > .validationErrorMarker:focus
+{
+  outline: none;
```

Comment:
TODO: need to review if we use to "correct" optiosn. Probably but needs review.

Reply:
A bit unsure of what you mean but I copied this from the following `Common.css` rule:

```css
input:not(:disabled).remotion-themed:focus,
textarea:not(:disabled).remotion-themed:focus,
select:not(:disabled).remotion-themed:focus,
button:not(:disabled).remotion-themed:focus,
.remotion-themed input:not(:disabled):focus,
.remotion-themed textarea:not(:disabled):focus,
.remotion-themed select:not(:disabled):focus,
.remotion-themed button:not(:disabled):focus
{
  outline: none;
  background-color: var(--remotion-themed-background-color-focus);
  border: var(--remotion-themed-border-focus);
  box-shadow: var(--remotion-themed-box-shadow-focus);
}
```